### PR TITLE
Git label commenter

### LINKED ISSUE #20: [CI] add label commenter workflow
**Description:**

This repository should have the label commenter workflow.
It would run whenever an issue/pr is labelled with one of the targeted labels and comment down any relevant information.

**Contributor Reference:**

The workflow would use: https://github.com/peaceiris/actions-label-commenter

A sample workflow is defined in: https://github.com/meshery/meshery/blob/master/.github/workflows/label-commenter.yml.

It would also need a sample configuration which is defined in: https://github.com/meshery/meshery/blob/master/.github/label-commenter-config.yml

**NOTE:**

- In the above configuration https://github.com/meshery/meshery/blob/88d1c7c559259775ce2affab2a15a32c673c3f52/.github/label-commenter-config.yml#L26-L41 are specific to Meshery and won't be needed on every repository.
- Do comment in PR if you find any label is missing for the repo.

### PR DESCRIPTION
**Pull request by monalisha31**
Signed-off-by: Monalisha Ojha <monalishaojha974@gmail.com>

**Description**

This PR fixes #20 

**Notes for Reviewers**

Added configuration for:
- ```issue/remind```
- ```issue/dco```
- ```duplicate```
- ```invalid```
- ```proposal```
- ```wontfix```

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
-  [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by warunicorn19** [comment]
hey @monalisha31 , looks like you forgot to Sign-off your commit, you could follow: https://github.com/layer5io/linkerd-service-mesh-workshop/pull/21/checks?check_run_id=3661063206 or if you don't have a sign-off set, you could give this a read https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin

File: .github/label-commenter-config.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+comment:
+
```
Comment (amino19) [inline .github/label-commenter-config.yml:2]:
```suggestion
```
Not reqd. an empty line between L1 & L3, pl remove [L2](https://github.com/layer5io/linkerd-service-mesh-workshop/blob/c8a1da20465e15f3e98a2796cff88220e6cf1a10/.github/label-commenter-config.yml#L2)

File: .github/label-commenter-config.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+comment:
+ 
+  footer: "\
+    ---\n\n
+    > &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Be sure to [join the community](http://slack.layer5.io), if you haven't yet and please leave a :star: [star on the project](../stargazers) :smile:
+    "
+
+labels:
+
```
Comment (amino19) [inline .github/label-commenter-config.yml:9]:
```suggestion
```
Pl remove [L9](https://github.com/layer5io/linkerd-service-mesh-workshop/blob/c8a1da20465e15f3e98a2796cff88220e6cf1a10/.github/label-commenter-config.yml#L9)

**Comment by monalisha31** [comment]
@warunicorn19 Thanks for letting me know. There was some error before while signing off the commits. Now, the checks are passed.

**Review by amino19 — CHANGES_REQUESTED** [review]
@monalisha31: Actually, there is no need to trigger `github-actions`  for further labels.
Do you mind remove [these](https://github.com/layer5io/linkerd-service-mesh-workshop/blob/c8a1da20465e15f3e98a2796cff88220e6cf1a10/.github/label-commenter-config.yml#L30-L55) lines? :slightly_smiling_face:

**Comment by amino19** [comment]
@warunicorn19: We need to create a label for `issue/dco`. :rocket:

**Comment by warunicorn19** [comment]
@amino19 done!

File: .github/label-commenter-config.yml (hunk 1 above)
Comment (monalisha31) [reply .github/label-commenter-config.yml]: Done

File: .github/label-commenter-config.yml (hunk 2 above)
Comment (monalisha31) [reply .github/label-commenter-config.yml]: Done

File: .github/label-commenter-config.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        body: "🚨 Alert! Git Police! We couldn’t help but notice that one or more of your commits is missing a sign-off. _A what?_ A commit sign-off (your email address).\n\n
+         To amend the commits in this PR with your signoff using the instructions provided in the DCO check above. \n\n
+         To configure your dev environment to automatically signoff on your commits in the future, see [these instructions](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)."
+        action: open
+      pr:
+        body: "🚨 Alert! Git Police! We couldn’t help but notice that one or more of your commits is missing a sign-off. _A what?_ A commit sign-off (your email address).\n\n
+         To amend the commits in this PR with your signoff using the instructions provided in the DCO check above. \n\n
+         To configure your dev environment to automatically signoff on your commits in the future, see [these instructions](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)."
+        action: open
+
```
Comment (amino19) [inline .github/label-commenter-config.yml:28]:
```suggestion
```
@monalisha31: Thanks for your efforts, just one minor change [here](https://github.com/layer5io/linkerd-service-mesh-workshop/blob/95285309f89818c1e0eb5f52bd6bf90a698ffa1c/.github/label-commenter-config.yml#L28), need to remove it.

**Review by amino19 — CHANGES_REQUESTED** [review]

**Review by warunicorn19 — APPROVED** [review]
LGTM🎉, Thanks @monalisha31

**Pull request merged**